pkg/profile/artifact.go

```diff
@@ -68,21 +61,23 @@ func (p *Profile) createHelmRelease(ctx context.Context) error {
 			Namespace: namespace,
 		},
 		TypeMeta: metav1.TypeMeta{
-			Kind:       helmReleaseKind,
-			APIVersion: helmReleaseAPIVersion,
+			Kind:       helmv2.HelmReleaseKind,
+			APIVersion: helmv2.GroupVersion.String(),
 		},
 		Spec: helmv2.HelmReleaseSpec{
 			Chart: helmv2.HelmChartTemplate{
 				Spec: helmv2.HelmChartTemplateSpec{
 					// TODO obvs don't rely on index 0
 					Chart: p.definition.Spec.Artifacts[0].Path,
 					SourceRef: helmv2.CrossNamespaceObjectReference{
-						Kind:      gitRepositoryKind,
+						Kind:      sourcev1.GitRepositoryKind,
 						Name:      p.makeGitRepoName(),
 						Namespace: namespace,
 					},
 				},
 			},
+			Values:     p.subscription.Spec.Values,
+			ValuesFrom: p.subscription.Spec.ValuesFrom,
```

Comment:
can you update the `artifact_test.go` to test these fields are set?

Reply:
Done